src/components/Comments.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <div class="mb-5" v-if="get(comments, 'length', 0)">
+    <div id="title" class=" font-medium text-indigo-600 pb-2">
+      <i class="el-icon-s-comment"></i>
+      {{ $t('comments.title') }}
+    </div>
+    <div
+      v-for="(comment, index) in comments"
+      :key="comment.id"
+      class="py-3 grid grid-cols-3 sm:grid-cols-4 md:grid-cols-8 border-b"
+      :class="index === comments.length - 1 ? 'border-none pb-0' : ''"
+    >
+      <div id="user-area" class="h-full col-span-1">
+        <img class="w-full rounded-full" :src="getAvatar(comment)" />
+      </div>
+      <div
+        id="comment-area"
+        class="pl-4 col-span-full col-start-2 place-content-start"
+      >
+        <div class="font-medium text-indigo-500 flex justify-between">
+          <span>
+            @{{
+              get(comment, 'relationships.creator[0].first_name', 'No name')
```

Comment:
let's create a computed prop because the code is the same and only get the names